src/server/YoolooServer.java

```diff
@@ -102,7 +112,45 @@ public void startServer() {
 			serverAktiv = false;
 			e1.printStackTrace();
 		}
+		restartSession(543210);
+	}
+	
+	public void sendCheatMessageToAllClients(YoolooClientHandler handlerToExpell)
+	{
+		clientHandlerList.remove(handlerToExpell);
+		ServerMessage notificationForPlayerCheating = new ServerMessage(ServerMessageType.SERVERMESSAGE_NOTIFY_CHEAT,
+				ClientState.CLIENTSTATE_NULL,
+				ServerMessageResult.SERVER_MESSAGE_RESULT_OK);
+	   for(int i = 0; i < clientHandlerList.size(); i++)
+	   {
+		   socketUtils.sendSerialized(clientHandlerList.get(i).getSocket(), notificationForPlayerCheating);
+	   }
+	}
+	
+	public synchronized void restartSession(int code)
+	{
+		if(code == 543210)
+		{
+		try {
+			Thread.sleep(1500);
+		} catch (InterruptedException e) {
+			e.printStackTrace();
+		}
+		for(YoolooClientHandler handler : clientHandlerList)

```

Comment:
Um zu vermeiden, dass sich Spieler doppelt anmelden können, habe ich in #2 hab ich den Typen von `clientHandlerList` auf `LinkedHashMap<String, YoolooClientHandler>` abgeändert.
Deshalb ist es nicht mehr möglich mittels einfachem schleifen-Konstrukt über die Liste zu iterieren, die `for` müsste so aussehen:
```java
for(Entry<String, YoolooClientHandler> ent : this.clientHandlerList.entrySet()) {
YoolooClientHandler handler = ent.getValue();
...
```